### PR TITLE
Automatic version bumping

### PR DESCRIPTION
**Pull request by pun-ky**

### CODE REVIEW

File: src/main/kotlin/com/neva/gradle/fork/config/rule/CopyTemplateFilesRule.kt [hunk 1]
```diff
@@ -12,19 +11,20 @@ class CopyTemplateFilesRule(config: Config, private val files: Map<String, Strin
...
     val regularFile = File(config.templateDir, templateName)
     return listOf(pebFile, regularFile).firstOrNull { it.exists() }
-      ?: throw ForkException("Fork template file does not exist: $pebFile")
   }
 
   private fun registerPromptsForTemplateFiles() {
     files.keys.forEach { templateName ->
       val templateFile = findTemplateFile(templateName)
-      val template = FileHandler(config, templateFile).read()
+      if (templateFile != null) {
```
Comment (pun-ky) [inline src/main/kotlin/com/neva/gradle/fork/config/rule/CopyTemplateFilesRule.kt]: fix for running just `sh gradlew tasks` with no tpl file for props task

File: src/main/kotlin/com/neva/gradle/fork/gui/PropertyDialog.kt [hunk 2]
```diff
@@ -31,18 +31,20 @@ class PropertyDialog(private val config: Config) {
...
   private var pathButton = JButton().apply {
-    text = "Pick a path"
-    addActionListener {
-      if (fileChooser.showOpenDialog(this) == JFileChooser.APPROVE_OPTION) {
-        fieldFocused!!.document.insertString(
-          fieldFocused!!.caretPosition,
-          fileChooser.selectedFile.absolutePath.replace("\\", "/"),
-          null
-        )
+    if (config.definedProperties.any { it.type == PropertyType.PATH || it.type == PropertyType.URI }) {
```
Comment (pun-ky) [inline src/main/kotlin/com/neva/gradle/fork/gui/PropertyDialog.kt]: path picker displayed only if there are properties with desired types

File: src/main/kotlin/com/neva/gradle/fork/ForkExtension.kt [hunk 3]
```diff
@@ -24,37 +22,24 @@ open class ForkExtension(val project: Project, val props: PropsExtension) {
       ?: throw ForkException("Fork configuration '$name' is yet not defined.")
   }
 
-  fun config(configurer: Closure<*>) {
-    config { ConfigureUtil.configure(configurer, this) }
-  }
-
-  fun config(configurer: Config.() -> Unit) {
+  fun config(configurer: Action<in SourceTargetConfig>) {
```
Comment (mierzwid) [inline src/main/kotlin/com/neva/gradle/fork/ForkExtension.kt]: LIKE

File: src/main/kotlin/com/neva/gradle/fork/config/Config.kt [hunk 4]
```diff
@@ -185,14 +189,13 @@ abstract class Config(val fork: ForkExtension, val name: String) {
   private fun promptPostProcess() {
     definedProperties.forEach { property ->
       if (property.type == PropertyType.PASSWORD) {
-         prompts[property.name]?.apply { value = fork.props.encryptor.encrypt(value) }
+        prompts[property.name]?.apply { value = fork.props.encryptor.encrypt(value) }
```
Comment (mierzwid) [inline src/main/kotlin/com/neva/gradle/fork/config/Config.kt]: Will that fix #22 ?

File: src/main/kotlin/com/neva/gradle/fork/gui/PropertyDialog.kt (hunk 2 above)
Comment (mierzwid) [reply src/main/kotlin/com/neva/gradle/fork/gui/PropertyDialog.kt]: NICE

**Pull request merged**